installDAQmx.py

```diff
@@ -0,0 +1,81 @@
+import urllib.request
```

Comment:
Do we need this if we already have the version in pyproject.toml to call?

Reply:
We dont need this file, I had copied for testing initial. I have deleted it